spec/models/user_spec.rb

```diff
@@ -7,7 +7,7 @@
     FactoryBot.create_list(:transaction, 10, user: user, amount: 10)
   end
 
-  context '.user_balance' do
+  context 'when checking user_balance' do
```

Comment:
@sinpai why not `.user_balance`?

Reply:
Rubocop rspec suggested that all context's should start with "when, with, without"

Reply:
Should I leave it like rspec suggests?

Reply:
@sinpai try `describe`. Is it work?

Reply:
@sinpai if not - let's discuss it in `#general` channel